packages/notebook-extension/src/tool-widgets/activeCellToolWidget.tsx

```diff
@@ -0,0 +1,110 @@
+/*
+ * Copyright (c) Jupyter Development Team.
+ * Distributed under the terms of the Modified BSD License.
+ */
+
+import React from 'react';
+import { FieldProps } from '@rjsf/utils';
+import { IEditorLanguageRegistry } from '@jupyterlab/codemirror';
+import { INotebookTracker, NotebookTools } from '@jupyterlab/notebook';
+import { ISharedText } from '@jupyter/ydoc';
+import { PanelLayout, Widget } from '@lumino/widgets';
+import { CodeCellModel, ICellModel, InputPrompt } from '@jupyterlab/cells';
+import { Debouncer } from '@lumino/polling';
+
+namespace Private {
+  /**
+   * Custom active cell field options.
+   */
+  export interface IOptions {
+    /**
+     * The tracker to the notebook panel.
+     */
+    tracker: INotebookTracker;
+
+    /**
+     * Editor languages registry
+     */
+    languages: IEditorLanguageRegistry;
+  }
+}
+
+/**
+ * The active cell field, displaying the first line and execution count of the active cell.
+ *
+ * ## Note
+ * This field does not work as other metadata form fields, as it does not update metadata.
+ */
+export class ActiveCellTool extends NotebookTools.Tool {
+  constructor(options: Private.IOptions) {
+    super();
+    const { languages } = options;
+    this._tracker = options.tracker;
+
+    this.addClass('jp-ActiveCellTool');
```

Comment:
Should all caps global const instead of inline CSS class string be used here?

Please see `shell.ts` for reference:
https://github.com/jupyterlab/jupyterlab/blob/82e1a74b61a41fbe5d65e8542053b749cdfe6ea0/packages/application/src/shell.ts#L39